src/main/groovy/com/palantir/gradle/javadist/tasks/CopyLauncherBinariesTask.groovy

```diff
@@ -0,0 +1,31 @@
+package com.palantir.gradle.javadist.tasks
+
+import com.palantir.gradle.javadist.JavaDistributionPlugin
+import org.gradle.api.file.FileCopyDetails
+import org.gradle.api.file.RelativePath
+import org.gradle.api.tasks.Copy
+
+class CopyLauncherBinariesTask extends Copy {
+    CopyLauncherBinariesTask() {
+        group = JavaDistributionPlugin.GROUP_NAME
+        description = "Creates go-java-launcher binaries."
+
+        project.afterEvaluate {
```

Comment:
yep, tasks include a reference to the project. that's typically how you bootstrap task config without an explicit `configure()` method